src/Model.php

```diff
@@ -1898,23 +1881,10 @@ protected function _rawInsert($m, $row)
         $m->reload_after_save = false;
         $m->unload();
 
-        if (!is_array($row)) {
-            $row = [$m->title_field => $row];
-        }
-
         // Find any row values that do not correspond to fields, and they may correspond to
         // references instead
         $refs = [];
         foreach ($row as $key => $value) {
-            // no field exists
```

Comment:
I don't see reason to remove this block. It's not related to default title_field or CSV and it's actually used in atk4\data\Reference\HasMany.

Reply:
It is actually related with CSV - see the explode there :)